hamocc/mo_read_oafx.F90

```diff
@@ -268,10 +286,28 @@ subroutine get_oafx(kpie,kpje,kplyear,kplmon,omask,oafx)
   !--------------------------------
   ! Scenarios of constant fluxes
   !--------------------------------
-  if( trim(oalkscen)=='const_0p14' .or. trim(oalkscen)=='const_0p56' ) then
+  if( trim(oalkscen)=='const' ) then
       
     oafx(:,:) = oalkflx(:,:)
 
+  !--------------------------------
+  ! Scenario of ramping-up fluxes
+  !--------------------------------
+  elseif(trim(oalkscen)=='ramp' ) then
+
+    if(kplyear.lt.ramp_start ) then
+      oafx(:,:) = 0.0
+    elseif(kplyear.ge.ramp_end ) then
+      oafx(:,:) = oalkflx(:,:)
+    else
+      current_day = (kplyear-ramp_start)*365.+nday_of_year
```

Comment:
Fortran compilers will probably handle this, but first you declare `current_day` as `int` and in l. 303 you write now 365. (real number) instead of 365 (integer).
So, I guess, I would either declare it as real above or remove the `.`, while casting all the stuff to float in the next line (where you can the use `365.`).

Reply:
Hi @jmaerz, good point thanks. It will be clearer despite having the compiler handling it.
if I leave `current_day` declared as `int`, just to keep it intuitive, are you fine with the below?
```
      current_day = (kplyear-ramp_start)*365+nday_of_year
      oafx(:,:) = oalkflx(:,:) * (current_day / ((ramp_end-ramp_start)*365.))
```
I hesitate using the Fortran intrinsic procedure `FLOAT(foo)`.

Reply:
I am fine with this (casting will be done internally). Out of curiosity: any reason not to use `float()`?

Reply:
No specific reason. That was my doubt. If FLOAT is used, I presume that there is no need to use a `.` to 365 knowing that it will be explicitely casted.

Let's fully agree :) Are you fine with this one?

`oafx(:,:) = oalkflx(:,:) * FLOAT( current_day / ((ramp_end-ramp_start)*365) )`

Reply:
Erh, now I am getting uncertain how Fortran will/would handle this - first making an integer division and then turning the truncated integer value into a float? Just stick with  
```oafx(:,:) = oalkflx(:,:) * current_day / ((ramp_end-ramp_start)*365.)```
which should conform to Fortran standards.